include/RED4ext/DynArray.hpp

```diff
@@ -306,12 +311,7 @@ struct DynArray
     uint32_t CalculateGrowth(uint32_t aNewSize)
     {
         uint32_t geometric = capacity + (capacity / 2);
-        if (geometric < aNewSize)
-        {
-            return aNewSize;
-        }
-
-        return geometric;
+        return std::max(aNewSize, geometric);
```

Comment:
Stupid Windows headers....

```suggestion
        return (std::max)(aNewSize, geometric);
```